service/src/main/java/bio/terra/cbas/controllers/RunsApiController.java

```diff
@@ -22,6 +27,25 @@ public RunsApiController(CromwellServerConfiguration cromwellConfig) {
     this.cromwellConfig = cromwellConfig;
   }
 
+  @Override
+  public ResponseEntity<RunLogResponse> getRun() {
+    List<LogRunRequest> runs = new ArrayList<>();
+    String runId = UUID.randomUUID().toString();
+    String name = "CBAS";
+    Date now = new Date();
+
+    runs.add(
+        (new LogRunRequest()
+            .runId(runId)
+            .state(RunState.UNKNOWN)
+            .workflowUrl("urlHere")
```

Comment:
(Just to confirm) is this an intentional placeholder for something that can't be done yet? (Same question for "params" below)

Reply:
Yes, definitely an intentional placeholder. The next ticket will be the actual wiring to Cromwell.